README.md

```diff
@@ -1 +1,8 @@
-# vscode-datavirt-extension-pack
\ No newline at end of file
+# VS Code - DataVirt Extension Pack
```

Comment:
maybe better to use the display name "Extension Pack for Data Virtualization by Red Hat"

Reply:
done